modules/ROOT/pages/getting-started.adoc

```diff
@@ -21,13 +21,13 @@ include::{partialsdir}/registering-a-mobile-app.adoc[leveloffset=1]
 
 {product-name} provide commonly required features for mobile app development.
 
-This section introduces the procedures for using {product-name} by guiding you through the process using the Identity Management service. For a full list of available services, see xref:con_services.adoc[Mobile Services]
+This section introduces the procedures for using {product-name} by guiding you through the process using the Identity Management service. For a full list of available services see xref:con_services.adoc[Mobile Services]
```

Comment:
@pmacko1 - based on what I see in the IBM style guide - that comma should be there

Reply:
what... i don't remember posting this...